gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/service/ServiceDistributionPluginTests.groovy

```diff
@@ -497,8 +497,10 @@ class ServiceDistributionPluginTests extends GradleTestSpec {
         def classpathJar = file('dist/service-name-0.0.1/service/lib/').listFiles()
                 .find({ it.name.endsWith("-manifest-classpath-0.0.1.jar") })
         classpathJar.exists()
-        readFromZip(classpathJar, "META-INF/MANIFEST.MF")
-                .contains('Class-Path: guava-19.0.jar produces-manifest-') // etc
+        String fromZip = readFromZip(classpathJar, "META-INF/MANIFEST.MF")
+        fromZip.contains('Class-Path:')
+        fromZip.contains('guava-19.0.jar')
+        fromZip.contains('produces-manifest-')
```

Comment:
but there is no `main` directory in the distribution, right? it's just the jars.